.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,52 @@
+name: Publish image to GitHub Container Registry
+
+on:
+  push:
+    branches: ["main"]
+
+jobs:
+  publish-image:
+    runs-on: ubuntu-latest
```

Comment:
Can we lock this to a version, eg. 22.04

Reply:
Great catch. I'm fixing this.